src/extlibs/hrvo/vector2.h

```diff
@@ -295,6 +295,19 @@ inline float det(const Vector2 &vector1, const Vector2 &vector2)
     return vector1.getX() * vector2.getY() - vector1.getY() * vector2.getX();
 }
 
+/**
+ * Computes the dot product of two vectors
+ *
+ * @param vector_left  The left hand side vector of dot product
+ * @param vector_right  The right hand side vector of dot product
+ * @return  The dot product of the two vectors.
+ */
+inline float dotProduct(const Vector2 &vector_left, const Vector2 &vector_right)
```

Comment:
can we add a TODO to link to #2371

Reply:
Added a TODO to the top of the file that the entire file should be removed with that ticket.